src/main/java/sdmExam/Board.java

```diff
@@ -0,0 +1,34 @@
+package sdmExam;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Board {
+    private List<Cell> _plays = new ArrayList<>();
```

Comment:
rename _plays to cells (let's try to be consistent with the notation: use camel case notation)